ui/store/actions.ts

```diff
@@ -5605,3 +5607,32 @@ export async function multichainUpdateBalance(
 export async function multichainUpdateBalances(): Promise<void> {
   return await submitRequestToBackground<void>('multichainUpdateBalances', []);
 }
+
+export async function getLastInteractedConfirmationInfo(): Promise<
+  LastInteractedConfirmationInfo | undefined
+> {
+  return await submitRequestToBackground<void>(
+    'getLastInteractedConfirmationInfo',
+  );
+}
+
+export async function setLastInteractedConfirmationInfo(
```

Comment:
This and the above actions aren't related to tracing.

Reply:
Removed in 0ace1c1ad3